tests/reflection/binding-gl.hlsl

```diff
@@ -0,0 +1,27 @@
+//TEST:REFLECTION:-profile ps_4_0 -target spirv
+
+// Confirm that we can generate reflection info for arrays
```

Comment:
Comment should maybe be updated for what this is now testing. :)

Maybe call the file `explicit-binding-vk-gl.hlsl` to capture the spirit?